solo/base/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN curl -o $SKYDNS_PLUGIN_DEB -L $SKYDNS_PLUGIN_DEB_URI \
     && rm $SKYDNS_PLUGIN_DEB
 
 # Install Go (from the official golang Dockerfile)
-ENV GOLANG_VERSION 1.6
+ENV GOLANG_VERSION 1.8.1
```

Comment:
Needed to be able to build the latest skydns. Didn't want to figure out a way to fix the version of skydns. https://github.com/spotify/helios/issues/901